src/libmeasurement_kit/nettests/meek_fronting.cpp

```diff
@@ -0,0 +1,24 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include <measurement_kit/nettests.hpp>
+#include <measurement_kit/ooni.hpp>
+
+namespace mk {
+namespace nettests {
+
+MeekFrontingTest::MeekFrontingTest() : BaseTest() {
+    runnable.reset(new MeekFrontingRunnable);
+    runnable->test_name = "meek_fronting";
```

Comment:
The correct test_name is: `meek_fronted_requests`. TBH I never really liked this name that much, but it's what we have 🤕